homeassistant/components/nws_alerts/const.py

```diff
@@ -0,0 +1,4 @@
+"""Constants for the NWS Alerts integration."""
+
+DOMAIN = "nws_alerts"
+API_ENDPOINT = "https://api.openweathermap.org/data/2.5/onecall?lat={lat}&lon={lon}&exclude=current,minutely,hourly,daily&appid={api_key}"
```

Comment:
This is the same endpoint that the [OpenWeatherMap](https://www.home-assistant.io/integrations/openweathermap/) integration users. 

https://github.com/csparpa/pyowm/blob/0474b61cc67fa3c95f9e572b96d3248031828fce/pyowm/weatherapi25/weather_manager.py#L502